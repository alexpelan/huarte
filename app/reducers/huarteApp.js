/* eslint-disable no-case-declarations */
import Common from '../util/Common';

const defaultState = {
  seasons: {},
  games: {},
  currentGameId: undefined,
  seasonsLoaded: false,
  seasonsLoadedTime: undefined,
  error: {
    isError: false,
    errorText: '',
  },
};

function huarteApp(state = defaultState, action) {
  let newState;
  switch (action.type) {
    case 'REQUEST_GAME':
      newState = Object.assign({}, state);
      return newState;
    case 'RECEIVE_GAME':
      newState = Object.assign({}, state);
      newState.games[action.gameId].jeopardy = action.game.jeopardy;
      newState.games[action.gameId].double_jeopardy = action.game.double_jeopardy;
      newState.games[action.gameId].final_jeopardy = action.game.final_jeopardy;
      newState.games[action.gameId].currentRound = 'jeopardy';
      newState.games[action.gameId].score = 0;
      newState.games[action.gameId].numberCorrect = 0;
      newState.games[action.gameId].numberIncorrect = 0;
      newState.games[action.gameId].loaded = true;
      newState.games[action.gameId].timeLoaded = action.timeLoaded;
      return newState;
    case 'SELECT_GAME':
      newState = Object.assign({}, state);
      newState.currentGameId = parseInt(action.gameId, 10);
      return newState;
    case 'REQUEST_SEASONS':
      newState = Object.assign({}, state);
      return newState;
    case 'RECEIVE_SEASONS':
      newState = Object.assign({}, state);
      newState.seasons = action.seasons;
      newState.seasonsLoaded = true;
      newState.seasonsLoadedTime = action.timeLoaded;
      return newState;
    case 'REQUEST_GAME_LIST':
      newState = Object.assign({}, state);
      return newState;
    case 'RECEIVE_GAME_LIST':
      newState = Object.assign({}, state);
      const seasonToUpdate = newState.seasons.find(season => season.seasonId === action.seasonId);
      Object.keys(action.games).forEach((gameId) => {
        newState.games[gameId] = action.games[gameId];
        seasonToUpdate.games.push(gameId);
      });
      seasonToUpdate.gamesLoaded = true;
      seasonToUpdate.timeLoaded = action.timeLoaded;
      return newState;
    case 'SELECT_QUESTION':
      newState = Object.assign({}, state);
      newState.games[state.currentGameId][action.round]
        .categories[action.categoryIndex]
        .clues[action.clueIndex].isCompleted = true;

      const modifiedCategory = newState.games[state.currentGameId][action.round]
        .categories[action.categoryIndex];
      let isCategoryCompleted = true;
      modifiedCategory.clues.forEach((clue) => {
        if (!clue.isCompleted) {
          isCategoryCompleted = false;
        }
      });

      if (isCategoryCompleted) {
        newState.games[state.currentGameId][action.round].categories[action.categoryIndex]
          .isCompleted = true;
      }

      return newState;
    case 'UPDATE_SCORE':
      newState = Object.assign({}, state);
      let wasCorrect = false;
      if (action.delta > 0) {
        wasCorrect = true;
      }

      Common.saveStatistics(wasCorrect, action.delta);

      const currentScore = state.games[state.currentGameId].score;
      newState.games[state.currentGameId].score = currentScore + action.delta;

      if (wasCorrect) {
        const newNumberCorrect = newState.games[state.currentGameId].numberCorrect + 1;
        newState.games[state.currentGameId].numberCorrect = newNumberCorrect;
      } else {
        const newNumberIncorrect = newState.games[state.currentGameId].numberIncorrect + 1;
        newState.games[state.currentGameId].numberIncorrect = newNumberIncorrect;
      }
      return newState;
    case 'NEXT_ROUND':
      newState = Object.assign({}, state);
      newState.games[state.currentGameId].currentRound = action.nextRound;
      return newState;
    case 'BID_FINAL_JEOPARDY':
      newState = Object.assign({}, state);
      newState.games[state.currentGameId].final_jeopardy.categories[0].value = `$${action.bid}`;
      return newState;
    case 'GLOBAL_SET_ERROR':
      newState = Object.assign({}, state);
      newState.error.isError = true;
      newState.error.errorText = action.error;
      return newState;
    case 'GLOBAL_DISMISS_ERROR':
      newState = Object.assign({}, state);
      newState.error.isError = false;
      newState.error.errorText = '';
      return newState;
    default:
      return state;
  }
}

export {
  defaultState,
  huarteApp,
};
