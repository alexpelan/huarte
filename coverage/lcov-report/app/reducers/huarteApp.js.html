<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for app/reducers/huarteApp.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../prettify.css" />
    <link rel="stylesheet" href="../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../index.html">All files</a> / <a href="index.html">app/reducers</a> huarteApp.js
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">45.83% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>33/72</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">30.43% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>7/23</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">75% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>3/4</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">45.83% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>33/72</span>
      </div>
    </div>
  </div>
  <div class='status-line low'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37
38
39
40
41
42
43
44
45
46
47
48
49
50
51
52
53
54
55
56
57
58
59
60
61
62
63
64
65
66
67
68
69
70
71
72
73
74
75
76
77
78
79
80
81
82
83
84
85
86
87
88
89
90
91
92
93
94
95
96
97
98
99
100
101
102
103
104
105
106
107
108
109
110
111
112
113
114
115
116
117
118
119
120</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">11x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-yes">10x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-yes">242x</span>
<span class="cline-any cline-yes">242x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">3x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import Common from "../util/Common";
&nbsp;
let defaultState = {
    seasons: {},
    games: {},
    currentGameId: undefined,
    seasonsLoaded: false,
    seasonsLoadedTime: undefined,
    error: {
      isError: false,
      errorText: ""
    }
};
&nbsp;
function huarteApp(state = defaultState, action) {
  var newState;
  switch(action.type) {
    case "REQUEST_GAME":
      newState = Object.assign({}, state);
      return newState; 
    case "RECEIVE_GAME":
      newState = Object.assign({},state);
      newState.games[action.gameId].jeopardy = action.game.jeopardy;
      newState.games[action.gameId].double_jeopardy = action.game.double_jeopardy;
      newState.games[action.gameId].final_jeopardy = action.game.final_jeopardy;
      newState.games[action.gameId].currentRound = "jeopardy";
      newState.games[action.gameId].score = 0;
      newState.games[action.gameId].numberCorrect = 0;
      newState.games[action.gameId].numberIncorrect = 0;
      newState.games[action.gameId].loaded = true;
      newState.games[action.gameId].timeLoaded = action.timeLoaded;
      return newState;
    case "SELECT_GAME":
      newState = Object.assign({}, state);
      newState.currentGameId = parseInt(action.gameId);
      return newState;
<span class="branch-3 cbranch-no" title="branch not covered" >    case "REQUEST_SEASONS":</span>
<span class="cstat-no" title="statement not covered" >      newState = Object.assign({}, state);</span>
<span class="cstat-no" title="statement not covered" >      return newState;</span>
    case "RECEIVE_SEASONS":
      newState = Object.assign({}, state);
      newState.seasons = action.seasons;
      newState.seasonsLoaded = true;
      newState.seasonsLoadedTime = action.timeLoaded;
      return newState;
<span class="branch-5 cbranch-no" title="branch not covered" >    case "REQUEST_GAME_LIST":</span>
<span class="cstat-no" title="statement not covered" >      newState = Object.assign({}, state);</span>
<span class="cstat-no" title="statement not covered" >      return newState;</span>
    case "RECEIVE_GAME_LIST":
      newState = Object.assign({}, state);
      const seasonToUpdate = newState.seasons.find((season) =&gt; {
        return season.seasonId === action.seasonId;
      });
      Object.keys(action.games).forEach((gameId) =&gt; {
        newState.games[gameId] = action.games[gameId];
        seasonToUpdate.games.push(gameId); 
      });
      seasonToUpdate.gamesLoaded = true;
      seasonToUpdate.timeLoaded = action.timeLoaded;
      return newState;
<span class="branch-7 cbranch-no" title="branch not covered" >    case "SELECT_QUESTION":</span>
<span class="cstat-no" title="statement not covered" >      newState = Object.assign({}, state);</span>
<span class="cstat-no" title="statement not covered" >      newState.games[state.currentGameId][action.round].categories[action.categoryIndex].clues[action.clueIndex].isCompleted = true;</span>
     
      const modifiedCategory = <span class="cstat-no" title="statement not covered" >newState.games[state.currentGameId][action.round].categories[action.categoryIndex];</span>
      let isCategoryCompleted = <span class="cstat-no" title="statement not covered" >true;</span>
<span class="cstat-no" title="statement not covered" >      modifiedCategory.clues.forEach(<span class="fstat-no" title="function not covered" >(c</span>lue) =&gt; {</span>
<span class="cstat-no" title="statement not covered" >        if (!clue.isCompleted) {</span>
<span class="cstat-no" title="statement not covered" >          isCategoryCompleted = false;</span>
        }
      });
&nbsp;
<span class="cstat-no" title="statement not covered" >      if (isCategoryCompleted) {</span>
<span class="cstat-no" title="statement not covered" >        newState.games[state.currentGameId][action.round].categories[action.categoryIndex].isCompleted = true;</span>
      }
&nbsp;
<span class="cstat-no" title="statement not covered" >      return newState;</span>
<span class="branch-8 cbranch-no" title="branch not covered" >    case "UPDATE_SCORE":</span>
<span class="cstat-no" title="statement not covered" >      newState = Object.assign({}, state);</span>
      let wasCorrect = <span class="cstat-no" title="statement not covered" >false;</span>
<span class="cstat-no" title="statement not covered" >      if (action.delta &gt; 0) {</span>
<span class="cstat-no" title="statement not covered" >        wasCorrect = true;</span>
      } 
&nbsp;
<span class="cstat-no" title="statement not covered" >      Common.saveStatistics(wasCorrect, action.delta);</span>
&nbsp;
      var currentScore = <span class="cstat-no" title="statement not covered" >state.games[state.currentGameId].score;</span>
<span class="cstat-no" title="statement not covered" >      newState.games[state.currentGameId].score = currentScore + action.delta;</span>
&nbsp;
<span class="cstat-no" title="statement not covered" >      if (wasCorrect) {</span>
<span class="cstat-no" title="statement not covered" >        newState.games[state.currentGameId].numberCorrect = newState.games[state.currentGameId].numberCorrect + 1;</span>
      } else {
<span class="cstat-no" title="statement not covered" >        newState.games[state.currentGameId].numberIncorrect = newState.games[state.currentGameId].numberIncorrect + 1;</span>
      }
<span class="cstat-no" title="statement not covered" >      return newState;</span>
<span class="branch-9 cbranch-no" title="branch not covered" >    case "NEXT_ROUND":</span>
<span class="cstat-no" title="statement not covered" >      newState = Object.assign({}, state);</span>
<span class="cstat-no" title="statement not covered" >      newState.games[state.currentGameId].currentRound = action.nextRound;</span>
<span class="cstat-no" title="statement not covered" >      return newState;</span>
<span class="branch-10 cbranch-no" title="branch not covered" >    case "BID_FINAL_JEOPARDY":</span>
<span class="cstat-no" title="statement not covered" >      newState = Object.assign({}, state);</span>
<span class="cstat-no" title="statement not covered" >      newState.games[state.currentGameId].final_jeopardy.categories[0].value = "$" + action.bid;</span>
<span class="cstat-no" title="statement not covered" >      return newState;</span>
<span class="branch-11 cbranch-no" title="branch not covered" >    case "GLOBAL_SET_ERROR":</span>
<span class="cstat-no" title="statement not covered" >      newState = Object.assign({}, state);</span>
<span class="cstat-no" title="statement not covered" >      newState.error.isError = true;</span>
<span class="cstat-no" title="statement not covered" >      newState.error.errorText = action.error;</span>
<span class="cstat-no" title="statement not covered" >      return newState;</span>
<span class="branch-12 cbranch-no" title="branch not covered" >    case "GLOBAL_DISMISS_ERROR":</span>
<span class="cstat-no" title="statement not covered" >      newState = Object.assign({}, state);</span>
<span class="cstat-no" title="statement not covered" >      newState.error.isError = false;</span>
<span class="cstat-no" title="statement not covered" >      newState.error.errorText = "";</span>
<span class="cstat-no" title="statement not covered" >      return newState;</span>
    default:
      return state;
  }
&nbsp;
}
&nbsp;
export default huarteApp;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Sun Jul 23 2017 17:38:32 GMT-0400 (EDT)
</div>
</div>
<script src="../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../sorter.js"></script>
</body>
</html>
