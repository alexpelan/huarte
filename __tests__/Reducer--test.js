import 'react-native';
// Note: test renderer must be required after react-native.
import { defaultState, huarteApp } from '../app/reducers/huarteApp';

test('reducer tests', () => {
  expect(huarteApp(defaultState, 'REQUEST_GAME')).toMatchSnapshot();
  expect(huarteApp(defaultState, 'RECEIVE_GAME')).toMatchSnapshot();
  expect(huarteApp(defaultState, 'SELECT_GAME')).toMatchSnapshot();
  expect(huarteApp(defaultState, 'REQUEST_SEASONS')).toMatchSnapshot();
  expect(huarteApp(defaultState, 'RECEIVE_SEASONS')).toMatchSnapshot();
  expect(huarteApp(defaultState, 'REQUEST_GAME_LIST')).toMatchSnapshot();
  expect(huarteApp(defaultState, 'RECEIVE_GAME_LIST')).toMatchSnapshot();
  expect(huarteApp(defaultState, 'SELECT_QUESTION')).toMatchSnapshot();
  expect(huarteApp(defaultState, 'UPDATE_SCORE')).toMatchSnapshot();
  expect(huarteApp(defaultState, 'NEXT_ROUND')).toMatchSnapshot();
  expect(huarteApp(defaultState, 'BID_FINAL_JEOPARDY')).toMatchSnapshot();
  expect(huarteApp(defaultState, 'GLOBAL_SET_ERROR')).toMatchSnapshot();
  expect(huarteApp(defaultState, 'GLOBAL_DISMISS_ERROR')).toMatchSnapshot();
});
