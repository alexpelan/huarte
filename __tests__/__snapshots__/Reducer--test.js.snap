// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reducer tests 1`] = `
Object {
  "currentGameId": undefined,
  "error": Object {
    "errorText": "",
    "isError": false,
  },
  "games": Object {},
  "seasons": Object {},
  "seasonsLoaded": false,
  "seasonsLoadedTime": undefined,
}
`;

exports[`reducer tests 2`] = `
Object {
  "currentGameId": undefined,
  "error": Object {
    "errorText": "",
    "isError": false,
  },
  "games": Object {},
  "seasons": Object {},
  "seasonsLoaded": false,
  "seasonsLoadedTime": undefined,
}
`;

exports[`reducer tests 3`] = `
Object {
  "currentGameId": undefined,
  "error": Object {
    "errorText": "",
    "isError": false,
  },
  "games": Object {},
  "seasons": Object {},
  "seasonsLoaded": false,
  "seasonsLoadedTime": undefined,
}
`;

exports[`reducer tests 4`] = `
Object {
  "currentGameId": undefined,
  "error": Object {
    "errorText": "",
    "isError": false,
  },
  "games": Object {},
  "seasons": Object {},
  "seasonsLoaded": false,
  "seasonsLoadedTime": undefined,
}
`;

exports[`reducer tests 5`] = `
Object {
  "currentGameId": undefined,
  "error": Object {
    "errorText": "",
    "isError": false,
  },
  "games": Object {},
  "seasons": Object {},
  "seasonsLoaded": false,
  "seasonsLoadedTime": undefined,
}
`;

exports[`reducer tests 6`] = `
Object {
  "currentGameId": undefined,
  "error": Object {
    "errorText": "",
    "isError": false,
  },
  "games": Object {},
  "seasons": Object {},
  "seasonsLoaded": false,
  "seasonsLoadedTime": undefined,
}
`;

exports[`reducer tests 7`] = `
Object {
  "currentGameId": undefined,
  "error": Object {
    "errorText": "",
    "isError": false,
  },
  "games": Object {},
  "seasons": Object {},
  "seasonsLoaded": false,
  "seasonsLoadedTime": undefined,
}
`;

exports[`reducer tests 8`] = `
Object {
  "currentGameId": undefined,
  "error": Object {
    "errorText": "",
    "isError": false,
  },
  "games": Object {},
  "seasons": Object {},
  "seasonsLoaded": false,
  "seasonsLoadedTime": undefined,
}
`;

exports[`reducer tests 9`] = `
Object {
  "currentGameId": undefined,
  "error": Object {
    "errorText": "",
    "isError": false,
  },
  "games": Object {},
  "seasons": Object {},
  "seasonsLoaded": false,
  "seasonsLoadedTime": undefined,
}
`;

exports[`reducer tests 10`] = `
Object {
  "currentGameId": undefined,
  "error": Object {
    "errorText": "",
    "isError": false,
  },
  "games": Object {},
  "seasons": Object {},
  "seasonsLoaded": false,
  "seasonsLoadedTime": undefined,
}
`;

exports[`reducer tests 11`] = `
Object {
  "currentGameId": undefined,
  "error": Object {
    "errorText": "",
    "isError": false,
  },
  "games": Object {},
  "seasons": Object {},
  "seasonsLoaded": false,
  "seasonsLoadedTime": undefined,
}
`;

exports[`reducer tests 12`] = `
Object {
  "currentGameId": undefined,
  "error": Object {
    "errorText": "",
    "isError": false,
  },
  "games": Object {},
  "seasons": Object {},
  "seasonsLoaded": false,
  "seasonsLoadedTime": undefined,
}
`;

exports[`reducer tests 13`] = `
Object {
  "currentGameId": undefined,
  "error": Object {
    "errorText": "",
    "isError": false,
  },
  "games": Object {},
  "seasons": Object {},
  "seasonsLoaded": false,
  "seasonsLoadedTime": undefined,
}
`;
